script/Ciel/Day 5/05_01_CIEL05_2_1.txt

```diff
@@ -88,33 +90,34 @@ But my eyeballs are still intact, as they haven't sustained any injuries.
 [sha:e7339fa32f4eab9cb35189e4107a9e9ffc0901cc]{
 -- Page 298, Offset 11657.
 -- 気炎を上げて吸血鬼を睨む。
-// Not sure boasting is the right word, 気炎を上げる
-I stare at the boasting vampire.
+I glare at the gloating vampire.
 }
 [sha:4f6a99d0a2c0b6902f9f499aec81bb3283e04cd8]{
 -- Page 298, Offset 11658.
 -- しかし『線』は消失していた。
-However, his 'lines' have disappeared.
+But the 'lines' have faded from my sight.
 }
 [sha:101c5be36e341794d312096c9c07d9dbe8c435f9]{
 -- Page 298, Offset 11659.
 -- ナイフを持つ指も砕けて散っていた。
-The fingers holding the knife crumble and scatter away.
+The fingers I had wrapped around the knife are crumbling and flaking away.
 }
 [sha:50b2fac6e85cfb3e7c1f243b89aaacadc11ae9d3]{
 -- Page 298, Offset 11660.
 -- ―――がしゃり、と。
-———Crack.
+———%{i}Crack%{/i}.
 }
 [sha:920c4f94eff3076b66fa3a05a1b0f5da7252509c]{
 -- Page 298, Offset 11661.
 -- 硝子の彫像が割れるような音が、鼓膜からではなく骨を伝わって、脳内に響きわたる。
-A sound, akin to a glass sculpture shattering, is perceived, not from my eardrums but through my bones, and echoes in my brain.
+// Are these sentences as heinously run-on to JP readers
```

Comment:
I'm obviously not a native speaker but it doesn't really, like, bother me? Maybe it's precisely because I'm non-native 🤷

Reply:
It doesn't stick out to me either on the JP side (obviously same caveat). Could just be that the different sentence ordering lends itself better to accumulating details?